routes/items/item_response.js

```diff
@@ -40,4 +40,7 @@ const whereCompleted = query => {
 const respondWithItems = (response, user) => ({ items, tree }) =>
   response.render( 'items/index', { user, items, tree: tree.children() })
 
-module.exports = { allItemsQuery, filteredItemsQuery, respondWithItems }
+const respondWithItemsApi = ( response, user ) => ({ items, tree }) =>
```

Comment:
I think we have an opportunity to create a more general function here:
```js
const respondWithItems = (user, callback) => ({ items, tree }) =>
  callback({ user, items, tree: tree.children() })
```
Now you can call this from the existing route with:
```js
respondWithItems( user, data => response.render( 'items/index', data ) )
```
And from your endpoint as:
```js
respondWithItems( user, data => response.json( data ) )
```